### ISSUE TITLE
Misaligned segmentation masks

### ISSUE THREAD

**Issue by kazuto1011** [kazuto1011/deeplab-pytorch#51]
Downsampling by PyTorch nearest interpolation in ```main.py``` results in misaligned masks. That's going to be replaced with Pillow nearest interpolation. Codes and pre-trained models will be updated sometime soon.

**Comment by kazuto1011** [comment]
FYI: 1/16 downsampling with various frameworks.
So far this repository has been used the right 3 methods.
![figure_1](https://user-images.githubusercontent.com/9032347/52703547-ec3b1780-2fc1-11e9-8ff8-e74d9cb8c611.png)
![figure_2](https://user-images.githubusercontent.com/9032347/52703559-f2c98f00-2fc1-11e9-9373-ac4389cb5a6b.png)
![figure_3](https://user-images.githubusercontent.com/9032347/52703566-f5c47f80-2fc1-11e9-833d-fe9f876fa94e.png)
